src/main.py

```diff
@@ -1,11 +1,29 @@
 import sys
+import os
+from Board import initBoard, getBoardFromConfig, displayBoard, shoot
+from constants import UNICODE_FOR_A_CHAR
+from Game import switchPlayer, initGame
+from utils import isValid
+from Config import getPlayersConfig
+import ui
 
-from Board import initBoard, getBoardFromConfig, displayBoard
+config1, config2 = getPlayersConfig()
 
-playerConfig = sys.argv[1].split("=")[1]
+boards, currentPlayer = initGame(config1, config2, 10)
+hit = 0
 
-with open("./config/" + playerConfig, 'r') as f:
-    config = f.read().splitlines()
+while True:
```

Comment:
Tou main is so much better like this, nice refacto 

![giphy](https://user-images.githubusercontent.com/2212144/92574654-a6bafb00-f287-11ea-8241-3c9a9210f984.gif)

Reply:
Thanks xD